src/docfx/lib/ProcessUtility.cs

```diff
@@ -453,6 +462,23 @@ private static LockInfo FilterExpiredAcquirers(LockInfo lockInfo)
             return lockInfo;
         }
 
+        private static Task<(bool, string)> AcquireWithRetry(string lockName, Func<Task<(bool, string)>> acquireLock)
+        {
+            return RetryUntilSucceed(lockName, ex => ex is ApplicationException, ThrowOnNotAcquired);
```

Comment:
>ThrowOnNotAcquired [](start = 81, length = 18)

This name is a bit confusing, it's like a callback to invoke when acquireLock failed